docs/self-managed/identity/user-guide/connect-to-an-existing-keycloak.md

```diff
@@ -0,0 +1,68 @@
+---
+id: connect-to-an-existing-keycloak
+title: "Connect to an existing Keycloak instance"
+sidebar_label: "Connect to an existing Keycloak instance"
+---
+
+In this guide, we'll demonstrate how to connect the Identity component to your existing Keycloak instance.
+
+### Prerequisites
+
+- Access to your [Keycloak Admin Console](https://www.keycloak.org/docs/16.1/server_admin/#using-the-admin-console)
+- A basic understanding of [administering realms and clients](https://www.keycloak.org/docs/16.1/server_admin/#assembly-managing-clients_server_administration_guide) in Keycloak
+
+### Steps
+
+To connect Identity to an existing Keycloak instance, take the following steps:
+
+1. Log in to your Keycloak Admin Console.
+
+2. Select the realm you would like to connect Identity to. In our example, this is **Test Realm**
+
+![keycloak-admin-realm-select](img/keycloak-admin-realm-select.png)
+
+3. Select **Clients** in the navigation menu, and click the **Create** button to create a new client.
+
+![keycloak-admin-client-list](img/keycloak-admin-client-list.png)
+
+4. Enter the client ID and the URL of where your Identity instance will be hosted and click **Save**.
+   :::note What client ID should I use?
+   By default, Identity uses the Client ID `camunda-identity`, so we recommend using this too. If you choose a different client ID, this will need to be set in the Identity application [environment variables](/docs/self-managed/identity/deployment/configuration-variables.md).
+   :::
+
+![keycloak-admin-client-add](img/keycloak-admin-client-add.png)
+
+5. On the page for the client created, set the **Access Type** to `confidential`, **Service Accounts Enabled** to `ON`, and save your changes by clicking the **Save** button.
```

Comment:
```suggestion
5. On the page for the created client, set the **Access Type** to `confidential`, **Service Accounts Enabled** to `ON`, and save your changes by clicking the **Save** button.
```

Reply:
Based on the screenshot below, you may also want to include a note for keeping any other defaults.

Reply:
I'm wary about adding too many note blocks to the page as they are quite large, for now I think I'd like to leave this step without a note and hope that the implied leaving of other fields is received.

Reply:
Sounds good to me!